framework/common/CMakeLists.txt

```diff
@@ -46,7 +46,7 @@ endif()
 
 # Create a single common library
 add_library(common ${sources})
-set_target_properties(common PROPERTIES PREFIX "mapf" VERSION ${prplmesh_VERSION} SOVERSION ${prplmesh_VERSION_MAJOR})
+set_target_properties(common PROPERTIES PREFIX "libmapf" VERSION ${prplmesh_VERSION} SOVERSION ${prplmesh_VERSION_MAJOR})
```

Comment:
Should probably change the library name to `mapfcommon` and remove the custom prefix.

Reply:
Done